nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -609,123 +610,6 @@ def test_plot_connectome_exceptions():
                         **kwargs)
 
 
-def test_connectome_strength():
```

Comment:
See previous note about not removing these tests. You'll also want to add a test that the `DeprecationWarning` is correctly raised.